resin/exceptions.py

```diff
@@ -313,3 +313,63 @@ def __init__(self, required_version, current_version):
         self.code = 'UnsupportedFunction'
         self.exit_code = 1
         self.message = Message.SUPERVISOR_VERSION_ERROR.format(req_version=required_version, cur_version=current_version)
+
+
+class AmbiguousApplication(Exception):
+    """
+    Exception of Resin ambiguous application.
+
+    Args:
+        application (str): application name.
+
+    Attributes:
+        code (str): exception code.
+        exit_code (int): program exit code.
+        message (str): error message.
+
+    """
+
+    def __init__(self, application):
+        self.code = 'AmbiguousApplication'
+        self.exit_code = 1
```

Comment:
My bad since I don't have more than one exit code, exit code is reserved for future use and we will definitely add more exit code and definition for each later.